google-cloud-contrib/google-cloud-nio/src/main/java/com/google/cloud/storage/contrib/nio/testing/FakeStorageRpc.java

```diff
@@ -41,7 +41,7 @@
 import javax.annotation.concurrent.NotThreadSafe;
 
 /**
- * A bare-bones in-memory implementation of Storage, meant for testing.
+ * A bare-bones in-memory implementation of StorageRpc, meant for testing of CloudStorageFileSystem.
```

Comment:
Two things:

1. I don't think you need the "of CloudStorageFileSystem" here. It's OK to use this to test other things if you don't care about buckets.

2. I think this is a wrong place to document this; the class is package-private. I think it's better to document [this](https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/google-cloud-contrib/google-cloud-nio/src/main/java/com/google/cloud/storage/contrib/nio/testing/LocalStorageHelper.java) instead, since more people are likely to see it.